docs/source/conf.py

```diff
@@ -0,0 +1,152 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
```

Comment:
you probably need to import shimmingtoolbox here to fix https://github.com/shimming-toolbox/shimming-toolbox-py/pull/41#issuecomment-666517789

Reply:
🤦‍♂️ I had missed that earlier, thanks!